testplan/docs/testbed.md

```diff
@@ -32,15 +32,15 @@ Different clusters are used on different platforms. Hence the clusters are descr
 
 ### OnPrem-Rancher
 
-| K8S      | Master              | Worker Nodes                                  | Resources                     |
+| K8S      | Master              | Worker Nodes                                  | Notes - Cluster Used           |
 | -------- | ------------------- | --------------------------------------------- | ----------------------------- |
-| Cluster1 | 1 Node 16GB 2  vCPU | 3 Node - Each node 16GB 4 vCPU                | Director Onprem Install       |
-| Cluster2 | 1 Node 16GB 2  vCPU | 5 Nodes - Each node  32GB 8CPU, 6x100GB Disks | OpenEBS Install Director      |
-| Cluster3 | 1 Node 16GB 2  vCPU | 3 Nodes - Each node  16GB 8CPU, 3x100GB Disks | OpenEBS Install Helm          |
-| Cluster4 | 1 Node 16GB 2  vCPU | 1 Node - Each node 16GB 2 vCPU,               | Director Onprem Upgrade       |
-| Cluster5 | 1 Node 16GB 2  vCPU | 3 Nodes - Each node  32GB 2CPU, 3x100GB Disks | OpenEBS Install Upgrade       |
-| Cluster6 | 1 Node 16GB 2  vCPU | 3 Nodes - Each node  16GB 2CPU, 3x100GB Disks | OpenEBS Install Upgrade       |
-| Cluster7 | 1 Node 32GB, 42vCPU | 10Nodes, Each Node 32GB 6vCPU,  6x100GB SSDs  | **Workload Cluster  OpenEBS** |
+| Cluster1 | 1 Node, 16GB, 2 vCPU | 3 Node - Each node 16GB 4 vCPU                | Director Onprem Install       |
```

Comment:
I am still not able to understand Notes section.
Can we rename the `Cluster1` to Ran-1